ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractRetryingSwitchingPeerTask.java

```diff
@@ -136,7 +125,7 @@ private void refreshPeers() {
     // If we are at max connections, then refresh peers disconnecting one of the failed peers,
     // or the least useful
 
-    if (peers.peerCount() >= peers.getMaxPeers()) {
+    if (peers.peerCount() >= peers.getPeerLowerBound()) {
```

Comment:
should this be upperBound?

Reply:
I want to force Besu to actively search for new peers, and my understanding is that this happens when the number of peers is below the lower bound

Reply:
maybe add a comment (or edit the existing comment) because the code is now doing something slightly different